src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4873,6 +4873,15 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>tuples</value>
   </data>
   <data name="IDS_FeatureOutVar" xml:space="preserve">
-    <value>out var</value>
+    <value>out variable declaration</value>
+  </data>
+  <data name="ERR_OutVariableUsedInTheSameArgumentList" xml:space="preserve">
+    <value>Reference to variable '{0}' is not permitted in the same argument list.</value>
```

Comment:
Thanks, I'll add it. This error is reported only for an implicitly-typed error. I'll clarify the spec and adjust the message too.